examples/qadi_demo.py

```
@@ -18,9 +19,19 @@
 from mad_spark_alt.core import (
     IdeaGenerationRequest,
     SmartQADIOrchestrator,
+    RobustQADIOrchestrator,
     ThinkingMethod,
```

Ruff (0.11.9)

`mad_spark_alt.core.ThinkingMethod` imported but unused

Remove unused import

(F401)